backend/src/main/resources/application-dev.properties

```diff
@@ -17,6 +18,12 @@ aws.ses.secretKey=${AWS_SECRET_KEY}
 aws.ses.endpoint=${AWS_SES_ENDPOINT_HOST}
 aws.ses.region=${AWS_REGION}
 
+aws.s3.accessKey=${AWS_S3_ACCESS_KEY_ID}
+aws.s3.secretKey=${AWS_S3_SECRET_KEY}
```

Comment:
It should use the same credentials as SES.

In the AWS console we need to create an IAM role called "backend" with permissions for both S3 and SES, then create a user with that role and finally use the credentials of that user.